_layouts/bib.liquid

```diff
@@ -185,9 +185,15 @@
     <!-- Links/Buttons -->
     <div class="links">
       {% if entry.award %}
-        <a class="award btn btn-sm z-depth-0" role="button">
-          {%- if entry.award_name %}{{ entry.award_name }}{% else %}Awareded{% endif -%}
-        </a>
+        {% if entry.award contains '://' %}
```

Comment:
What if the text someone wrote contains a link inside? Maybe it is best to check if the link in not a url with some other checks, for instance, if it doesn't start with one of `http`, `https`, `ftp` and doesn't contain any spaces.